spearhead/assets/sass/navigation.scss

```diff
@@ -8,3 +8,16 @@ $navigation-min-break-point: 'laptop';
 $navigation-max-break-point: 'laptop-only';
 
 @import "../../../seedlet/assets/sass/components/navigation/style";
+
+.primary-navigation,
+.woo-navigation {
+	.sub-menu {
+		@include media( $navigation-min-break-point ) {
+			padding: calc(0.5 * var(--primary-nav--padding)) calc( 2 * var(--primary-nav--padding) );
```

Comment:
It is unfortunate that all of these styles need to be added to change one value.

Reply:
Yeah, there was a similar thing earlier with margins: https://github.com/Automattic/themes/pull/2681/files

I wonder if we could simplify these styles in Seedlet to make them easier to override in the child themes? cc @kjellr

Reply:
I timeboxed an exploration into simplifying the styles, and it deserves some deeper consideration than a quick refactor to make this change easier.

Going to merge this with a comment that we should keep use cases like this in mind when we get around to simplifying Seedlet's nav styles.